.gitignore

```diff
@@ -46,3 +46,4 @@ eclipse-build
 !/.settings/org.eclipse.jdt.ui.prefs
 
 bin/*
+.DS_Store
```

Comment:
Can we confirm that other OpenSearch projects have these?   I know some projects instead say "put this in your global git ignore file!"